mantidproject/mantid
# Added a deselect all elements button to elemental analysis gui

**Description of work.**
This PR adds a new button to the elemental analysis gui which deselects all elements that have been selected.


**To test:**

1. Open the elemental analysis gui.
2. Load 2695 and select any detectors.
3. select multiple elements from the periodic table on left of gui.
4. Click on Deselect All Elements button on top right of gui ,all line on plot should be removed and all elements previously selected will be unselected. 

Fixes #26750 . 





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
